examples/arm/qemu-shell.nix

```diff
@@ -4,8 +4,9 @@ let
 in
 pkgs.mkShell {
   LANG="C.UTF-8";
-  LD_LIBRARY_PATH = "${crossPkgs.buildPackages.gcc-unwrapped}/aarch64-unknown-linux-gnu/lib64";
+  LD_LIBRARY_PATH = "${crossPkgs.buildPackages.gcc-unwrapped.out.lib}/aarch64-unknown-linux-gnu/lib";
   buildInputs = [
     pkgs.qemu
+    pkgs.strace
```

Comment:
Is this still needed or was it only for debugging?

Reply:
Only for debugging.
```suggestion
```